gitignore.md

```diff
@@ -1,3 +0,0 @@
-# Add a .gitignore
-
-See https://raw.githubusercontent.com/renuo/rails-application-setup-guide/master/templates/.gitignore
```

Comment:
This section is not referenced in any chapter and the referenced file does not exist any more.